docs/fdc3-charter.md

```diff
@@ -36,7 +36,7 @@ The group's activities do not include:
 
 - Versions of the FDC3 specification following 2.1 and subsequent draft specifications are licensed under the [Community Specification License 1.0](https://github.com/finos/FDC3/blob/4ce90d45ca8e0c4f8f2c5bd73f51304278783d87/LICENSE.md)
```

Comment:
Should this point at head version of license?

Reply:
We prefer using hashes, as it's not mutable; even though we don't plan to change the license in FDC3, we thought it was a good practice to follow.